src/navigation/MainNavigator.tsx

```diff
@@ -14,6 +15,8 @@ import ActivationStack from "./ActivationStack"
 import SettingsStack from "./SettingsStack"
 import ModalStack from "./ModalStack"
 import Welcome from "../Welcome"
+import CallbackStack from "./CallbackStack"
```

Comment:
```suggestion
import CallbackStack from "./CallbackStack"

import { Headers } from "../styles"
```

Reply:
I wouldn't bother for this commit but usually we put styles and assets in their own group!